osgar/lib/quaternion.py

```diff
@@ -62,4 +62,12 @@ def from_axis_angle(axis, angle):
     qw = math.cos(angle / 2)
     return [qx, qy, qz, qw]
 
+
+def rotation_matrix(quaternion):
```

Comment:
I suppose that you was "inspired from somewhere", so I would add here link??

Reply:
It is still the same stuff: https://en.wikipedia.org/wiki/Conversion_between_quaternions_and_Euler_angles

Reply:
Maybe we should move the link from `euler_zyx` to the top of the file?

Reply:
OK